src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkControlV.java

```diff
@@ -125,6 +127,8 @@ private void tryPasteBlock(@Nonnull BlockMenu blockMenu) {
             targetBlock.setType(fetchedStack.getType(), true);
             if (SupportedPluginManager.getInstance().isMcMMO()) {
                 mcMMO.getPlaceStore().setTrue(targetBlock);
+            } else if (SupportedPluginManager.getInstance().isAureliumSkills()) {
```

Comment:
This code assumed McMMO and Aurelium cannot be on the same server together. Very likely of course but not impossible. Also now we have two of these can I suggest a public static method in the SupportedPluginManager called registerBlockPlacement that fires off two private methods doing the placements for mcMMO and Aurelius if they’re present?

Reply:
Done something like this.
It is ready for merge now 
Can always polish later.